### PR TITLE
E2E tests in Cypress and Playwright

### PR DESCRIPTION
**Pull request by BOHEUS**
Proof of Concept

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -2,4 +2,11 @@
 .DS_Store
 node_modules/
 # yarn is the recommended package manager across the project
-.package-lock.json
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (charlesBochet) [inline .gitignore:5]: Some remaining git conflicts!

File: cypress.config.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+const { defineConfig } = require("cypress");
```
Comment (charlesBochet) [inline cypress.config.js:1]: could we move that to it's own folder packages?
The vision on the repo is to have everything under packages (even front and server but that's another story)

File: playwright/example.spec.ts [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import { test, expect } from '@playwright/test';
```
Comment (charlesBochet) [inline playwright/example.spec.ts:1]: same for playwright!

**Review by charlesBochet — COMMENTED** [review]
Exciting, so that's a POC with both technologies and we choose after?

**Comment by BOHEUS** [comment]
Exactly like that

**Comment by charlesBochet** [comment]
@BOHEUS Thank you, could you take the comments into account ? we can merge and test from there :)

**Comment by charlesBochet** [comment]
@BOHEUS I'm closing this PR to prevent keeping staled PRs in the list, feel free to re-open it with new changes and will happily take another look :)

**Pull request closed without merging**